src/routes/quantity/add.ts

```diff
@@ -5,6 +5,7 @@ import { schema as siNumberSchema, SiNumber } from '../../models/si-number';
 interface Request {
   body: {
     name: string;
+    color?: string;
```

Comment:
It's a bit ambiguous for who we provide the interface: if it's for the backend, color is not optional as the default is used. If it's for the front-end, it should indeed be optional. I propose to aim it at the backend: that uses it right now and it only requires typescript front-ends using this version or newer to explicitly specify a color.

Reply:
Using the same rationale you might argue that a `.default(..)` in any model (in src/models, which are only used in responses from the server) will never be used, as the Interface provides it correctly as non-optional.

Reply:
I was thinking about back compatibility here, but have been mistaken with layer a bit and almost added redundant check to controller :)

default value already handles old clients.

so I've removed `?`